app/models/consultation.rb

```diff
@@ -51,17 +51,25 @@ def neighbour_letters_sent?
     neighbour_letters.sent.present?
   end
 
+  def neighbour_letter_header
+    I18n.t("neighbour_letter_header.#{planning_application.application_type.name}",
+           closing_date: end_date_from_now.to_date.to_fs(:day_month_year))
+  end
+
   def neighbour_letter_content
-    I18n.t("neighbour_letter_template",
-           received_at: planning_application.received_at.to_fs(:day_month_year_slashes),
-           expiry_date: planning_application.expiry_date.to_fs(:day_month_year_slashes),
+    I18n.t("neighbour_letter_template.#{planning_application.application_type.name}",
+           received_at: planning_application.received_at.to_date.to_fs(:day_month_year),
```

Comment:
```suggestion
           received_at: planning_application.received_at.to_date.to_fs,
```

`:day_month_year` is default.